src/fhir/1_Questionnaire-C-IDAS.json

```diff
@@ -0,0 +1,1433 @@
+              "url": "http://hl7.org/fhir/StructureDefinition/ordinalValue",
+              "valueDecimal": 2
+            }
+          ],
+          "valueCoding": {
+            "code": "LA6750-9",
+            "display": "Severe"
+          }
+        },
+        {
+          "extension": [
+            {
+              "url": "http://hl7.org/fhir/StructureDefinition/questionnaire-optionPrefix",
+              "valueString": "a"
+            },
+            {
+              "url": "http://hl7.org/fhir/StructureDefinition/ordinalValue",
+              "valueString": "a"
+            }
+          ],
+          "valueCoding": {
+            "code": "LA4489-6",
+            "display": "Unable to evaluate"
+          }
+        }
+      ]
+    },
+    {
+      "type": "choice",
+      "required": false,
+      "linkId": "cs-idas-6",
+      "text": "Multiple physical complaints",
+      "answerOption": [
+        {
+          "extension": [
+            {
+              "url": "http://hl7.org/fhir/StructureDefinition/questionnaire-optionPrefix",
+              "valueString": "0"
+            },
+            {
+              "url": "http://hl7.org/fhir/StructureDefinition/ordinalValue",
+              "valueDecimal": 0
+            }
+          ],
+          "valueCoding": {
+            "code": "LA9634-2",
+            "display": "Absent"
+          }
+        },
+        {
+          "extension": [
+            {
+              "url": "http://hl7.org/fhir/StructureDefinition/questionnaire-optionPrefix",
+              "valueString": "1"
+            },
+            {
+              "url": "http://hl7.org/fhir/StructureDefinition/ordinalValue",
+              "valueDecimal": 1
+            }
+          ],
+          "valueCoding": {
+            "code": "LA6752-5",
+            "display": "Mild or intermittent"
+          }
+        },
+        {
+          "extension": [
+            {
+              "url": "http://hl7.org/fhir/StructureDefinition/questionnaire-optionPrefix",
+              "valueString": "2"
+            },
+            {
+              "url": "http://hl7.org/fhir/StructureDefinition/ordinalValue",
+              "valueDecimal": 2
+            }
+          ],
+          "valueCoding": {
+            "code": "LA6750-9",
+            "display": "Severe"
+          }
+        },
+        {
+          "extension": [
+            {
+              "url": "http://hl7.org/fhir/StructureDefinition/questionnaire-optionPrefix",
+              "valueString": "a"
+            },
+            {
+              "url": "http://hl7.org/fhir/StructureDefinition/ordinalValue",
+              "valueString": "a"
+            }
+          ],
+          "valueCoding": {
+            "code": "LA4489-6",
+            "display": "Unable to evaluate"
+          }
+        }
+      ]
+    },
+    {
+      "type": "choice",
+      "required": false,
+      "linkId": "cs-idas-7",
+      "text": "Loss of interest - less involved in usual activities",
```

Comment:
There are a few minor differences between this assessment and the PDF they provided, but if this was the one they identified in the FHIR library I think we can dismiss those differences

Reply:
thank you @sierramatice I added the missing text from the PDF they provided.